src/components/StudyHeader/StudyHeader.js

```diff
@@ -26,9 +26,11 @@ const StudyHeader = ({
   return (
     <Container>
       <Header as="h1">{studyName}</Header>
-      <Header.Subheader as="h2" className="study--header-sub">
-        {shortName}
-      </Header.Subheader>
+      {shortName && (
+        <Header.Subheader as="h2" className="study--header-sub">
```

Comment:
why the class?

Reply:
There was some extra space before the copy study id button when there is no short name. This will remove those space